rasa/train.py

```diff
@@ -54,8 +67,10 @@ def train(
         domain: Path to the domain file.
         config: Path to the config for Core and NLU.
         training_files: Paths to the training data for Core and NLU.
-        output: Output path.
+        output_path: Output path.
         force_training: If `True` retrain model even if data has not changed.
+        fixed_model_name: Name of model to be stored.
+        uncompress: If 'True' the model will not be compressed.
```

Comment:
Not a 100% sure, but I think you need `
```suggestion
        uncompress: If `True` the model will not be compressed.
```